docker/python/paddle/cloud/dataset/uci_housing.py

```diff
@@ -0,0 +1,30 @@
+import paddle.v2.dataset.uci_housing as uci_housing
+import paddle.v2.dataset.common as common
+import os
+
+__all__=["train", "test", "fetch"]
+
+CURRENT_DATACENTER = os.getenv("CURRENT_DATACENTER")
+
+common.DATA_HOME = "/pfs/%s/public/dataset" % CURRENT_DATACENTER
```

Comment:
`public` is the public folder mount in Pod with *readOnly* access, and add a sub-folder `dataset` to storage all the datasets is help for adding others in the `public` folder. And also in PaddlePaddle dataset, the directory is `~/.cache/paddle/dataset/...`